src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -248,6 +248,30 @@ export class StudyViewPageStore {
                 this.updateChartStats();
             }
         });
+
+        // Include special charts into custom charts list
+       SpecialCharts.forEach(chartMeta => {
```

Comment:
`SpecialCharts` looks like a class/type name rather than a constant, using all capitals for constants is a more common practice, i.e: `SPECIAL_CHARTS`.